core/src/layers/observe/metrics.rs

```diff
@@ -663,18 +661,17 @@ impl<A: Access, I: MetricsIntercept> LayeredAccess for MetricsAccessor<A, I> {
             });
 
         self.interceptor
-            .observe(labels.clone(), MetricValue::OperationExecuting(-1));
+            .observe(labels, MetricValue::OperationExecuting(-1));
         res
     }
 
     async fn read(&self, path: &str, args: OpRead) -> Result<(RpRead, Self::Reader)> {
         let labels = MetricLabels::new(self.info.clone(), Operation::Read.into_static());
 
-        let start = Instant::now();
-
         self.interceptor
             .observe(labels.clone(), MetricValue::OperationExecuting(1));
 
+        let start = Instant::now();
```

Comment:
Hi, it's intended to add start before observing `OperationExecuting`.

Reply:
But we did this like below, so should it also be adjusted?

```
        self.interceptor
            .observe(labels.clone(), MetricValue::HttpExecuting(1));
        let start = Instant::now();
```

Reply:
I'm guessing only `MetricsHttpFetcher` implemented in this way? I prefer to adapt `MetricsHttpFetcher` instead.

Reply:
> I'm guessing only `MetricsHttpFetcher` implemented in this way? 

Before this PR, Yes

Reply:
So, what is the final decision?